app/src/main/java/com/graeberj/taskman/auth/domain/models/LoggedInUser.kt

```diff
@@ -1,7 +1,7 @@
 package com.graeberj.taskman.auth.domain.models
 
 data class LoggedInUser(
-    val token: String,
-    val userId: String,
-    val fullName: String
+    val token: String = "",
+    val userId: String = "",
+    val fullName: String = ""
 )
```

Comment:
I'd recommend to avoid assigning such empty strings as default values. What you want to model is that these fields may not be existent at all times (such as when a user isn't logged in), but that would rather be modeled by a this whole class being set to null.
So something like `var loggedInUser: LoggedInUser? = null`

As soon as a user is logged in, all these fields must exist, so they should be non-nullable and especially not set to empty strings (which quickly gets confusing because you model what nullability means without getting nullable behavior)